src/main/kotlin/dsm/wemeet/global/s3/S3Util.kt

```
@@ -0,0 +1,95 @@
+            .key(fileName)
+            .contentType(MediaType.ALL_VALUE)
+            .acl(ObjectCannedACL.PRIVATE)
+            .contentLength(file.size)
+            .build()
+
+        try {
+            s3Client.putObject(
+                putObjectRequest,
+                RequestBody.fromInputStream(file.inputStream, file.size)
+            )
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)